.travis.yml

```diff
@@ -35,8 +35,10 @@ install:
     #   except scipy and install pandas with --no-deps
     # - pandas >= 0.20.0 is required since commit 01669f2024a7bffe47cceec0a0fd845f71b6f7cc
     #   (issue 702 : fixed bug when writing metadata using HDF format)
+    # - pydantic: cannot define numpy ndarray / pandas obj / LArray field with default value
```

Comment:
Are we blocked on an older version forever or is it a temporary problem?

Reply:
Hard to say. I opened an [issue ](https://github.com/samuelcolvin/pydantic/issues/2923) on the pydantic github repository. 
`pydantic` is still currently in active development but I don't how they manage their issues.